ts/nni_manager/training_service/reusable/environments/remoteEnvironmentService.ts

```diff
@@ -208,6 +208,33 @@ export class RemoteEnvironmentService extends EnvironmentService {
         this.remoteMachineMetaOccupiedMap.set(remoteEnvironment.rmMachineMeta, false);
     }
 
+    private async getScript(environment: EnvironmentInformation): Promise<string> {
+        const executor = await this.getExecutor(environment.id);
+        const logLevel = getLogLevel();
+        let script: string = environment.command;
+
+        if (executor.getIsWindows()) {
+            const prepare = `envs\\install_nni.cmd && mkdir envs\\${environment.envId} && cd envs\\${environment.envId}`;
+            const startrun = `cd .. && install_nni.cmd && cd ${environment.envId} && python -m nni.tools.trial_tool.trial_runner`;
+            const developingScript = "IF EXIST nni_trial_tool (ECHO \"nni_trial_tool exists already\") ELSE (mkdir nni_trial_tool && tar -xof ../nni_trial_tool.tar.gz -C ./nni_trial_tool) && pip3 install websockets";
+            
+            script = logLevel == "debug" ? `${prepare} && ${developingScript} && ${startrun}` : `${prepare} && ${startrun}`;
+        } else {
+            const prepare = `mkdir -p envs/${environment.envId} && cd envs/${environment.envId}`;
+            const startrun = `sh ../install_nni.sh && python3 -m nni.tools.trial_tool.trial_runner`;
+            const developingScript = "[ -d \"nni_trial_tool\" ] && echo \"nni_trial_tool exists already\" || (mkdir ./nni_trial_tool && tar -xof ../nni_trial_tool.tar.gz -C ./nni_trial_tool) && pip3 install websockets";
```

Comment:
Seems these linux commands already set in trialDispatcher.ts and could use original commands directly, only need to add windows comands in remoteEnvironmentService.ts?

Reply:
Right, Fixed.